### PR TITLE
chore: dont deploy to runtime

### PR DESCRIPTION
**Pull request by tripodsan**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -60,16 +58,14 @@ jobs:
           name: Link Sequences
           command: npm run deploy-routes
       - helix-post-deploy/monitoring:
-          targets: universal, aws
+          targets: aws
```
Comment (trieloff) [inline .circleci/config.yml:61]:
```suggestion
          targets: universal, aws
```

**Review by trieloff — CHANGES_REQUESTED** [review]
We should keep the gateway, even if we don't deploy to AIO anymore.

File: .circleci/config.yml (hunk 1 above)
Comment (tripodsan) [reply .circleci/config.yml:61]: none of our services is deployed to universal anymore. so I would remove it here as well.

Comment (trieloff) [reply .circleci/config.yml:61]: The published service URL and the service URL that is being used by other services is on anywhere.run, so the removal would prevent new functionality or bug fixes from being used.

**Pull request merged**